README.md

```diff
@@ -1,3 +1,5 @@
 # test-01
 
 jhhjkjhjkhkjs
+
+;wjkjewhkjeh
```

Comment:
The added lines seem to be random characters. Is there a specific purpose for this change? Please provide more context or remove them.